astropy/modeling/tests/test_parameters.py

```diff
@@ -32,7 +32,7 @@ class SetterModel(FittableModel):
     outputs = ('z',)
 
     xc = Parameter(default=1, setter=setter1)
-    yc = Parameter(default=1, setter=setter2)
+    #yc = Parameter(default=1, setter=setter2)
```

Comment:
Isn't the `setter2` functionality supported any more?

Reply:
This will require some thought and may require some change in usage. Previously most parameters were inextricably linked to models. To make them shareable, that linkage had to be broken, and the previous usage requires that linkage. I think it is possible to make this work without any changes. Tune in later for updates.

Reply:
fixed